robottelo/ui/discoveredhosts.py

```diff
@@ -100,32 +100,17 @@ def reboot_host(self, hostname):
                 'Could not find the selected discovered host "{0}"'
                 .format(hostname)
             )
-        drop_locator = locators['discoveredhosts.dropdown']
-        self.click((drop_locator % hostname))
-        reboot_locator = locators['discoveredhosts.reboot']
-        self.click((reboot_locator % hostname), wait_for_ajax=False)
+        self.click((locators['discoveredhosts.dropdown'] % hostname))
```

Comment:
These extra brackets are redundant, they're needed only with our old `strategy, value` approach to form a tuple, but as this PR is for 6.3 you can drop them.